cmds/cmd_version.py

```diff
@@ -80,7 +94,12 @@ async def print_version_info(message: discord.Message, args: List[str], client:
 
     fmt.write(f"\nBot Uptime: {getdelta(bot_start_time)}\n```")
 
-    await message.channel.send(fmt.getvalue())
+    content = fmt.getvalue()
+
+    if len(content) <= 2000:
+        await message.channel.send(content)
+    else:
+        await message.channel.send("ERROR: Exeeded discord message length limits, tell a developer to stop being lazy about rendering this")
```

Comment:
if I tell you here it kills the fun right?

Reply:
gonna wait till it actually becomes a problem to deal with it